schema/pod.go

```diff
@@ -129,16 +129,16 @@ func (al AppList) Get(name types.ACName) *RuntimeApp {
 // Mount describes the mapping between a volume and an apps
```

Comment:
Fixed this

On Fri, Aug 14, 2015 at 4:58 PM Jonathan Boulle <notifications@github.com>
wrote:

> In schema/pod.go
> <https://github.com/appc/spec/pull/475#discussion_r37129435>:
>
> > @@ -129,16 +129,16 @@ func (al AppList) Get(name types.ACName) *RuntimeApp {
> >  // Mount describes the mapping between a volume and an apps
>
> this too :|
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/spec/pull/475/files#r37129435>.
>